### ISSUE TITLE
Build errors when using gradle 3.6.1

### ISSUE THREAD

**Issue by myralchista** [haksist/DropdownTextView#11]

**Comment by myralchista** [comment]
when using new gradle version inside projects build gradle: classpath 'com.android.tools.build:gradle:3.6.1'

Errors when building project occurs: 
```
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml","position":{"startLine":6,"startColumn":4,"startOffset":267,"endLine":24,"endColumn":24,"endOffset":1208}}],"original":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/title_text (aka com.muraDev.psychotests:attr/title_text) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml","position":{"startLine":6,"startColumn":4,"startOffset":267,"endLine":24,"endColumn":24,"endOffset":1208}}],"original":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/content_text (aka com.muraDev.psychotests:attr/content_text) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml","position":{"startLine":6,"startColumn":4,"startOffset":267,"endLine":24,"endColumn":24,"endOffset":1208}}],"original":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/title_text_color (aka com.muraDev.psychotests:attr/title_text_color) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml","position":{"startLine":6,"startColumn":4,"startOffset":267,"endLine":24,"endColumn":24,"endOffset":1208}}],"original":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/title_text_color_expanded (aka com.muraDev.psychotests:attr/title_text_color_expanded) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml","position":{"startLine":6,"startColumn":4,"startOffset":267,"endLine":24,"endColumn":24,"endOffset":1208}}],"original":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/content_text_color (aka com.muraDev.psychotests:attr/content_text_color) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml","position":{"startLine":6,"startColumn":4,"startOffset":267,"endLine":24,"endColumn":24,"endOffset":1208}}],"original":"/home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/link_text_color (aka com.muraDev.psychotests:attr/link_text_color) not found.\n    ","tool":"AAPT"}

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Android resource linking failed
     /home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/title_text (aka com.muraDev.psychotests:attr/title_text) not found.
         
     /home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/content_text (aka com.muraDev.psychotests:attr/content_text) not found.
         
     /home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/title_text_color (aka com.muraDev.psychotests:attr/title_text_color) not found.
         
     /home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/title_text_color_expanded (aka com.muraDev.psychotests:attr/title_text_color_expanded) not found.
         
     /home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/content_text_color (aka com.muraDev.psychotests:attr/content_text_color) not found.
         
     /home/mrme/.gradle/caches/transforms-2/files-2.1/91316c6a9cec87f89ffe203bdb96d307/jetified-dropdown-textview-0.3.0/res/values/values.xml:7:5-25:25: AAPT: error: resource attr/link_text_color (aka com.muraDev.psychotests:attr/link_text_color) not found.
```
         
When clicking on the links of errors - it shows the file "values.xml" with "declare-styleable name="DropdownTextView" so it's releted to DropdownTextView module:

```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <dimen name="arrow_default_size">25dp</dimen>
    <dimen name="content_default_padding">12dp</dimen>
    <dimen name="panel_default_padding">12dp</dimen>
    <string name="app_name">DropdownTextView</string>
    <declare-styleable name="DropdownTextView">
        <attr name="title_text"/>
        <attr name="content_text"/>
        <attr format="reference" name="bg_drawable_regular"/>
        <attr format="reference" name="bg_drawable_expanded"/>
        <attr format="reference" name="arrow_drawable"/>
        <attr name="title_text_color"/>
        <attr name="title_text_color_expanded"/>
        <attr format="dimension" name="title_text_size"/>
        <attr format="reference" name="title_font"/>
        <attr format="dimension" name="content_text_size"/>
        <attr name="content_text_color"/>
        <attr name="link_text_color"/>
        <attr format="boolean" name="underline_link"/>
        <attr format="reference" name="content_font"/>
        <attr format="integer" name="expand_duration"/>
        <attr format="dimension" name="panel_padding"/>
        <attr format="dimension" name="content_padding"/>
    </declare-styleable>
</resources>
```

On older gradle 3.5.3 everything works ok.

**Comment by EsmaeelNabil** [comment]
same issue

**Comment by hakobast** [comment]
Fixed in version 0.3.1
